packages/react-charting/jest.config.js

```diff
@@ -19,7 +19,7 @@ function getEsmOnlyPackagesToCjsMapping() {
 const config = createConfig({
   setupFiles: ['./config/tests.js'],
   snapshotSerializers: ['@fluentui/jest-serializer-merge-styles', 'enzyme-to-json/serializer'],
-
+  setupFilesAfterEnv: ['@testing-library/jest-dom/extend-expect'],
```

Comment:
We should use unified setup for v8 packages:

```suggestion
  setupFilesAfterEnv: ['./config/tests.js'],  
```

any kind of package jest setup lives within `./config/tests.js` 
see https://github.com/microsoft/fluentui/blob/master/packages/react/jest.config.js#L4

Reply:
@Hotell, whenever I am trying to import in config/tests.js: require('@testing-library/jest-dom/extend-expect') I am getting this error: ReferenceError: expect is not defined. Please can you suggest what needs to be done in this case?

Reply:
Sreetama lets discuss about this issue.